backend/tests/homepageController.test.ts

```diff
@@ -233,6 +234,27 @@ describe('Homepage', () => {
       )
     })
 
+    it.skip('should render home page with the legacy book a prison visit task', async () => {
```

Comment:
did yo mean to put a `skip` here - what's the purpose of the test if it is skipped?

Reply:
Was just copying the existing PVB test and missed that. Will fix.